scipy/interpolate/_cubic.py

```diff
@@ -374,11 +375,12 @@ class Akima1DInterpolator(CubicHermiteSpline):
 
     Parameters
     ----------
-    x : ndarray, shape (m, )
+    x : ndarray, shape (npoints, )
         1-D array of monotonically increasing real values.
-    y : ndarray, shape (m, ...)
-        N-D array of real values. The length of ``y`` along the first axis
-        must be equal to the length of ``x``.
+    y : ndarray, shape (..., npoints, ...)
+        N-D array of real values. The length of ``y`` along the *first* axis
```

Comment:
Also the interpolation axis is not necessarily the first one.

Reply:
Fixed